index.js

```diff
@@ -16,25 +16,26 @@ const chalk = require('chalk');
 const buildToMatchSchema = require('./matchers/toMatchSchema');
 const buildToBeValidSchema = require('./matchers/toBeValidSchema');
 
-function matchersWithOptions(userOptions = {}) {
+function matchersWithOptions(userOptions = {}, afterAjvInitialized) {
```

Comment:
I think `extendAjv` would be a better variable name here (same applies to usage in matchers)

Reply:
Yes, much better naming suggestion!